src/StatsTools/reweighting.jl

```diff
@@ -164,3 +175,35 @@ end
 function mixed_estimator(df::DataFrame, h; kwargs...)
     return mixed_estimator(df.hproj, df.vproj, df.shift, h, df.dτ[end]; kwargs...)
```

Comment:
This is pretty cool! Can you add `hproj` and `vproj` keyword arguments for changing the column names in the DataFrame? You can run a computation multiple different `ProjectedEnergy`s and it would be cool to have this work with all of them. The name of `shift` can also change if you're running multiple replicas.

Another note: sometimes (below the plateau) `ratio_of_means` will error with a `QRFactorizationError` (I think). I don't know exactly how to reproduce it. It would be cool to have a way to avoid that (or we could use a `try ... catch` block).

Reply:
Thanks! Yes, I can do that later.

The `QRFactorizationError` sounds like it would come from `MonteCarloMeasurements`. I've never seen it though myself. Can you send a copy of the error message when it happens again? It's probably an issue that should be dealt with from MCM.

Reply:
@mtsch Have a look at [8b89bde](https://github.com/joachimbrand/Rimu.jl/pull/111/commits/8b89bdead7fd197877e1ff9fb3c362dd24a540e5), which resolves the `QRFactorizationError`.

We could do something similar for `Complex` vectors, but is a bit more complicated as any row and column with vanishing variance has to be removed from the covariance matrix.

Reply:
added keyword arguments for changing column names